src/core/variables.jl

```diff
@@ -84,6 +93,24 @@ struct AdditionalDeltaActivePowerDownVariable <: VariableType end
 
 struct SmoothACE <: VariableType end
 
+struct ActivePowerVariableThermal <: VariableType end
```

Comment:
We shouldn't need to do this. There has to be a better way to integrate with the existing infrastructure that won't require exploding in the number of variables we support